threefive/commands.py

```diff
@@ -37,6 +37,11 @@ def encode(self, nbin=None):
         nbin = self._chk_nbin(nbin)
         return nbin.bites
 
+    def from_xml(self, stuff):
```

Comment:
this should be base.py SCTE35Base class. It has the xml()

Reply:
Yeah, fair point - I should have spotted that. I'd have removed this but you already merged it :smile: